src/run_tribler.py

```diff
@@ -3,7 +3,9 @@
 import os
 import signal
 import sys
+import tkinter
```

Comment:
[Quick note while this is in WIP] `tkinter` is no longer a default package in Python, see:

https://github.com/Tribler/tribler/issues/3567
https://github.com/Tribler/tribler/issues/4799

The recommended approach is to use Qt for popups.

Reply:
It is a pity... Well, thank you for pointing this out.

Reply:
Fixed